.github/CONTRIBUTING.md

```diff
@@ -1,12 +1,24 @@
 
 ## Development install
 
-You should *always* use a virtual environment when developing software. Setup:
+### PyPI
+
+You can set up a development environment using PyPI.
 
 ```bash
-python3 -m venv .env
-. .env/bin/activate
-pip install -e .[dev]
+$ python -m venv .env
+$ source .env/bin/activate
+(.env)$ pip install -r requirements.txt
+(.env)$ pip install -e .
```

Comment:
This should remain `-e .[dev]`, and we should not add the requirements.txt. The requirements.txt is wrong, too, it should not install boost-histogram version right before you install a boost-histogram from source.

Reply:
Packages should not have a requirements.txt, see the discussion on scikit-hep/particle's recent issue.

Reply:
> we should not add the requirements.txt

Yes, packages should not have requirements.txt. But I think requirements.txt might be ok for development. See the [aghast #32](https://github.com/scikit-hep/aghast/pull/31), and these changes are made for CONTRIBUTING.md.

> The requirements.txt is wrong, too, it should not install boost-histogram version right before you install a boost-histogram from source.

If so, is it ok to remove boost-histogram requirement?

Reply:
Please make this dev-requirements.txt at least; I still rather not have this though since it can get out of sync with setup.py, while the `[dev]` cannot.

It is okay to assume developers know a bit more than normal users.

Reply:
All right. I will do what I did for aghast (adding <-test> suffix for requirements.txt and environment.yml).